tee-worker/cli/src/benchmark/direct_request_vc.rs

```diff
@@ -0,0 +1,159 @@
+use crate::{
+	command_utils::{get_shielding_key, get_worker_api_direct},
+	trusted_base_cli::commands::litentry::request_vc_subcommands::Command,
+	trusted_cli::TrustedCli,
+	trusted_command_utils::{get_identifiers, get_pair_from_str},
+	trusted_operation::send_direct_vc_request,
+	Cli, CliResult, CliResultOk,
+};
+use clap::Parser;
+use core::time::Duration;
+use ita_stf::{Getter, TrustedCall, TrustedCallSigned};
+use itc_rpc_client::direct_client::DirectClient;
+use itp_stf_primitives::{
+	traits::TrustedCallSigning,
+	types::{KeyPair, TrustedOperation},
+};
+use litentry_primitives::{RequestAesKey, ShardIdentifier, REQUEST_AES_KEY_LEN};
+use rayon::iter::{IntoParallelIterator, ParallelIterator};
+use sgx_crypto_helper::rsa3072::Rsa3072PubKey;
+use sp_core::Pair;
+use std::time::Instant;
+
+// Command to perform direct request_vc benchmarking. It creates a given number of threads and sends
+// `requests_per_thread` requests one by one recording time to get response. It uses Alice as
+// an IdGraph root. If bench case requires any identities to be linked, the preparation should be done
```

Comment:
Does it mean that the IDGraph can only be keyed by Alice? Like I have to link my identities to Alice and then request-vc under the name of Alice?

While this should be fine too, can we actually request vc with real IDGraph without Alice? Alice is used for signing only

Reply:
I've omitted account to whom the vc will be issued in order to simplify usage.

Yes, we can read it from cli args in order to have better flexibility.